src/OpenRiaServices.Hosting.AspNetCore/Framework/AspNetCore/OpenRiaServicesOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+#nullable enable
+
+namespace OpenRiaServices.Hosting.AspNetCore
+{
+    public sealed class OpenRiaServicesOptions
+    {
+        //public Action<Exception, HttpContext> OnError { get; set; }
```

Comment:
I think it would be good to be able to have IncludeExceptionMessageInFault a little bit more flexible 

One option is to return a string instead of a bool in the IncludeExceptionMessageInFault. 
But I like the idea to be able to configure the error a bit more and set the ErrorCode etc with a more flexible OnError.

AlwaysIncludeExceptionMessageInFault might be redundant in that case since you quite easily can configure the exception message to be passed on

Reply:
I will keep 2 variants, the exception handler does not allow setting stacktrace currently